### DISCUSSION TITLE
Child safety policy

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#58] [Qualified]
### What are some documents containing these terms?

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | Exam Success | [URL 🔗](https://www.examsuccess.com.au/info/child_safe_policy) | Child safe policy | 🇦🇺 |
| 2 | YouTube | [URL 🔗](https://support.google.com/youtube/answer/2801999?hl=en) | Child safety policy | 🌐 |
| 3 | CSIRO | [URL 🔗](https://www.csiro.au/en/about/Policies/Child-Safe-Policy) | Child safety policy | 🇦🇺 |
| 4 | Children International | [URL 🔗](https://www.children.org/child-protection-policy) | Child protection policy | 🇺🇸 |
| 5 | Wesley Mission | [URL 🔗](https://www.wesleymission.org.au/about-us/governance/privacy-and-policies/child-safe/) | Child safety policy | 🇦🇺 |
| 6 | UNIQ YOU | [URL 🔗](https://www.uniqyou.com.au/child-safety-policy/) | Child safety policy | 🇦🇺 |
| 7 | Meta | [URL 🔗](https://about.meta.com/actions/safety/onlinechildprotection/) | Online child protection | 🌐 |
| 8 | Meta | [URL 🔗](https://transparency.meta.com/policies/community-standards/child-sexual-exploitation-abuse-nudity/) | Child Sexual Exploitation, Abuse, and Nudity | 🇺🇸  |

### What are some alternative names for this type?

Child protection policy

### Who is the target of these terms?

end user

### What is the object of commitment in these terms?

protecting children from abuse and ensuring their safety

### Are there any relevant references for this type?

- [Children First Act 2015](https://www.irishstatutebook.ie/eli/2015/act/36/enacted/en/pdf)
- [Child Safe Standards](https://ccyp.vic.gov.au/child-safe-standards/)

**Comment by MattiSG** [comment]
- The YouTube example is a subset of its community guidelines, as [tracked in PGA](https://github.com/OpenTermsArchive/pga-versions/blob/main/YouTube/Community%20Guidelines.md#child-safety-policy).
- The Meta (n°7) example is a commercial / high-level document that does not constitute a valid example in my opinion.
- The precise Meta policy (n°8) is a subset of its community guidelines, as [tracked in PGA](https://github.com/OpenTermsArchive/pga-versions/blob/main/Facebook/Community%20Guidelines.md#child-sexual-exploitation-abuse-and-nudity).

It seems that most examples are found in Australia, as a consequence of its [child protection legislation](https://aifs.gov.au/resources/resource-sheets/australian-child-protection-legislation).

At this stage, I would consider that this type is specific to the Australian jurisdiction and would postpone its inclusion until partners are established within that jurisdiction to fully understand that topic.
